### PR TITLE
OCPBUGS-19806 updated CO release notes with known issue

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.11-4.13

Issue:
https://issues.redhat.com/browse/OCPBUGS-19806

Link to docs preview:
[CO 1.2.0 known issues (release notes)](https://file.rdu.redhat.com/antaylor/OCPBUGS-19806/security/compliance_operator/compliance-operator-release-notes.html#compliance-operator-1-2-0-known-issues)

Additional information:
None

### CODE REVIEW

**Comment by mkumku** [comment]
LGTM 10x!

**Review by rhmdnd — APPROVED** [review]
/lgtm

File: security/compliance_operator/compliance-operator-release-notes.adoc [hunk 1]
```diff
@@ -50,6 +50,11 @@ If your {product-title} environment contains existing `cis` and `cis-node` remed
 
 * Additional clarity for auditing security context constraints (SCCs) is now available for the `scc-limit-container-allowed-capabilities` rule.
 
+[id="compliance-operator-1-2-0-known-issues"]
+=== Known issues
+
+* When using the CIS {product-title} 4 Benchmark v1.4.0 profile, some controls might fail due to tightened permissions in the CIS profile than exists in {product-title}. For more information, see link:https://access.redhat.com/solutions/7024725[Solution article #7024725].
```
Comment (kalexand-rh) [inline security/compliance_operator/compliance-operator-release-notes.adoc:56]:
```suggestion
* When using the CIS {product-title} 4 Benchmark v1.4.0 profile, some controls might fail due to tighter permissions in the CIS profile than in {product-title}. For more information, see link:https://access.redhat.com/solutions/7024725[Solution article #7024725].
```
Have you sent this through CM?

Comment (sheriff-rh) [reply security/compliance_operator/compliance-operator-release-notes.adoc:56]: No, but I will! Thanks!

**Comment by sheriff-rh** [comment]
@xiaojiey can you review when you get a chance? Thanks!

**Review by rhdcaspin — APPROVED** [review]

**Comment by rhdcaspin** [comment]
Looks good

**Comment by sheriff-rh** [comment]
This has passed the change management process. Merging.

**Pull request merged**